app/lib/email.js

```diff
@@ -1,28 +1,18 @@
-const nodemailer = require('nodemailer');
-const moment = require('moment');
+
 const { EMAIL } = require('./constants');
 const logger = require('./logger');
+const moment = require('moment');
+const nodemailer = require('nodemailer');
 
 let transport = nodemailer.createTransport(EMAIL.CONFIGURATION);
 
-// Test code.
-// transport.sendMail({
-//   to: 'someone@example.com',
-//   subject: "Test " + moment().toISOString(),
-//   text: "Hi there!"
-// })
-
-//
-// By default, send emails to admins with standard message, from dunedb.
-//
-// Override any of these fields in the code
-module.exports = async function(arg) {
-  var data = {
-    from: EMAIL.FROM,
-    subject: "Dune DB Message " + moment().toISOString(),
-    to: EMAIL.ADMIN_RECIPIENTS,
-    ...arg
-  };
-  logger.info("sending email",data);
+module.exports = async function(arg)
+{
+  var data = {from   : EMAIL.FROM,
+              subject: "DUNE APA DB Message " + moment().toISOString(),
+              to     : EMAIL.ADMIN_RECIPIENTS,
+              ...arg};
+  
```

Comment:
Also here you are moving from a standard JavaScript style to an nonstandard one.  Note also that when we implement linting all this work would be automated.